.github/issue_template.md

```diff
@@ -0,0 +1,26 @@
+PLEASE FOLLOW THE CHECKLIST BELOW WHEN CREATING A NEW ISSUE. THE CHECKLIST IS
+FOR YOUR INFORMATION AN MUST BE REMOVED BEFORE SUBMITTING ISSUE.
+
+## Checklist
+
+- [ ] Is the issue marked with exactly one of the following labels
+
+    - Bug
+    - Feature
+    - Improvement
+    - infrastructure
+
+- [ ] If the issue reports a bug:
+
+    - [ ] Is it also marked with one of the `Impact - {Minor,Major,Critical}` labels?
+    - [ ] Does the description contain all necessary information to reproduce
+        the error?
+
+- [ ] Is the issue assigned to a milestone? This should be `next` unless you
+    know exactly what you are doing.
+
+- [ ] If the issue is a question or the start of a discussion, please add the
+    `Needs discussion` label.
+
+- [ ] If the issue tracks a long development or multiple underlying issues
+    please add the `Long Term` label.
```

Comment:
I honestly think we should not have `Long Term` issues, as they reduce the utility of all the other issues because they drown them out.

Reply:
We decided that we have some need for them and I do not want to open up that discussion again. And since we have them, they should be addressed in the checklist.

I do not think that this drowns them out since you can actually select only non-long-term issues by using the labels.

Reply:
I don't recall *us* deciding this but ok. We have the stale bot, which more or less goes exactly against having long term issues.

Reply:
I would argue we could/should use projects to track long term *issues* that span multiple individual actual issues. A bit like the `v1.0.0` project I opened recently.

Reply:
Yes, we did. During the workshop earlier this year.

Reply:
🤷‍♀️